packages/integrations/image/package.json

```diff
@@ -40,6 +40,7 @@
     "test": "mocha --exit --timeout 20000 test"
   },
   "dependencies": {
+    "@altano/tiny-async-pool": "^1.0.2",
```

Comment:
In case anyone is hesitant to take on [`@altano/tiny-async-pool`](https://www.npmjs.com/package/@altano/tiny-async-pool), a new dependency:

* The used export is super tiny: [315.0B gzipped](https://bundlephobia.com/package/@altano/tiny-async-pool@1.0.2)
* The package is a fork of [async-pool](https://github.com/rxaviers/async-pool) by Rafael Xavier de Souza. I left it MIT licensed and barely modified the code other than to modernize it (typescript, esm export, vitest, etc). [The original package](https://www.npmjs.com/package/tiny-async-pool) gets ~900K weekly downloads so it's mature code and should be fairly bug free.